### ISSUE TITLE
pagination when using collection

### ISSUE THREAD

**Issue by VickG** [Chumper/Datatable#288]
hello guys,

I just want to ask to be sure .. is it true that if I use the collection method, it will pull in all records and bypass pagination? VS using Query which will allow proper pagination?

please someone confirm.
Thanks

**Comment by VickG** [comment]
I have a  table I am pulling in to datatable.. it has about 1700 rows and it's currently pulling in all 1700 rows even thought my datatable view only shows 10 rows at a time.. I would like it to only pull 10 rows instead of the whole table data.

Is it true this is only possible with Query method??

I am currently using Collection Method

**Comment by Chumper** [comment]
Yes, that is correct, Collection will pull all data before it will evalueate and sort and do the pagination, thats why Query ius much faster.

What is your use case?
